crates/pop-cli/src/commands/up/mod.rs

```diff
@@ -168,7 +168,15 @@ mod tests {
 			Some(
 				DeploymentProvider::VARIANTS
 					.into_iter()
-					.map(|action| (action.name().to_string(), action.description().to_string()))
+					.map(|action| {
+						(
+							action.name().to_string(),
+							format!(
+								"{}",
+								style(format!("{}", action.base_url())).bold().underlined()
```

Comment:
The combination of description and URL didn’t look great (too long). Showing only the URL might be more helpful to the user.